src/index.ts

```diff
@@ -502,7 +439,7 @@ async function normalizeOpts(opts: any) {
         return 'esm';
       }
       return format;
-    }),
+    }) as [ModuleFormat, ...ModuleFormat[]],
```

Comment:
what is this?!

Reply:
haha, I had to [look up how to do this](https://stackoverflow.com/q/56006111/3431180).

This type means that there is "at least one" `ModuleFormat` in the array vs. plain `ModuleFormat[]` could mean no elements

Reply:
loll interesting.